### PR TITLE
Encyclopedia

### PR DESCRIPTION
**Pull request by 19ec94**
This PR will 
- fix the white screen appearance after 3D Text Font loading problem in finalMessage function call
- fix the "loading..." text in place of empty scoreboard table 
- fix the routing issue in the Signup component
- fix the Navbar rendering after Login and Signup
- add Dashboard to Navbar
- implement Logout functionality in Dashboard
- implement Delete Account functionality in Dashboard

### CODE REVIEW

File: src/components/Anatomy.jsx [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+  }, [url]);
+
+  return (
+    // Center component centers the text
+    <Center>
+      {/* Pass the loaded font to the component for rendering */}
+      <Text3DBase font={font} {...props} />
+    </Center>
+  );
+}
```
Comment (rmorasis) [inline src/components/Anatomy.jsx]: I am not really sure why you are changing the Text3D? 
I think it should work now without any changes required after I added the Inter_Bold.json font file yesterday to main.
Am I missing something here?

File: src/components/Anatomy.jsx [hunk 2]
```diff
@@ -288,39 +307,44 @@ export const AnatomyLevel1 = () => {
...
           bevelSize={0.04}
           bevelThickness={0.1}
           height={0.5}
-          lineHeight={0.5}
-          letterSpacing={-0.06}
+          lineHeight={1.0}
+          letterSpacing={0.06}
           size={1.5}
-          font="/Inter_Bold.json">
+          //font="/Inter_Bold.json"
```
Comment (rmorasis) [inline src/components/Anatomy.jsx]: I am not really sure why you are changing the Text3D? 
I think it should work now without any changes required after I added the Inter_Bold.json font file yesterday to main.
Am I missing something here?

**Review by rmorasis — COMMENTED** [review]
Encyclopedia works for me.

Like I commented, I don't see the need to change anything in Anatomy.jsx unless you can convince me otherwise.

File: src/components/Anatomy.jsx (hunk 1 above)
Comment (19ec94) [reply src/components/Anatomy.jsx]: Okay. I added this approach before merging the file "Inter_Bold.json in `public/` dirctory. I reverted the Anatomy.jsx back to your version. It should work now.

File: src/components/Anatomy.jsx (hunk 2 above)
Comment (19ec94) [reply src/components/Anatomy.jsx]: So, I reverted the Anatomy.jsx file to the main branch version.

**Review by rmorasis — APPROVED** [review]
LGTM

**Pull request merged**